modules/caddyhttp/templates/templates.go

```diff
@@ -120,6 +120,13 @@ func init() {
 // {{include "path/to/file.html" "arg1" 2 "value 3"}}  // with arguments
 // ```
 //
+// ##### `includeRaw`
```

Comment:
Needs a newline here I think

```suggestion
// ##### `includeRaw`

```

Reply:
Yep, I think so, but looking more at the format of the comments around it, shouldn't it be more like this?
```go
// ##### `includeRaw`
//
// Includes the contents of another file, without rendering it as a template.
//
// ```
// {{includeRaw "path/to/file.html"}}
// ```
//
```

Reply:
Yeah oops my bad, that's what I meant to suggest but I just didn't type the `//`

Reply:
That's what I figured, just wanted to make sure!